README.rst

```diff
@@ -23,6 +23,23 @@ The contents will be:
 - power colours
 - rms spectra
 
+Installation
+-------------
```

Comment:
I guess what Matteo meant to say was the option of installing dependencies with `pip install -r requirements.txt` and then run `python setup.py install.`

+1 for having `develop`, but I also think it should be in `How to contribute` page or somewhere.